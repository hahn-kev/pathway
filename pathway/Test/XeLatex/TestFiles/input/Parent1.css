
.xitem[lang='en']
{  color: orange;}

.main > .xitem{ color: red;}

.xitem + .xitem{ color: blue;}
.xitem + .xitem[lang='en']{ color: green;}
