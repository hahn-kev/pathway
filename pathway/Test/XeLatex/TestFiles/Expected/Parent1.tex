\documentclass[a4paper]{article} 
\usepackage{changepage}
\usepackage{float}
\usepackage{grffile}
\usepackage{graphicx}
\usepackage{amssymb}
\usepackage{fontspec}
\usepackage{fancyhdr}
\usepackage{multicol}
\usepackage{calc}
\usepackage{lettrine}
\usepackage[left=2cm,right=2cm,top=1.99cm,bottom=1.99cm,includeheadfoot]{geometry}

\parindent=0pt
\parskip=\medskipamount
\begin{document} 
\pagestyle{plain} 
\font\xitemen="Times New Roman":color=ffa500 at 12pt
\font\xitemmain="Times New Roman":color=ff0000 at 12pt
\font\xitemenxitem="Times New Roman":color=008000 at 12pt
\font\main="Times New Roman" at 12pt
\font\xitemenmain="Times New Roman":color=008000 at 12pt
\font\xitemxitemmain="Times New Roman":color=0000ff at 12pt

\pagestyle{fancy} 

\xitemenmain{sample text file }

\xitemxitemmain{sample text file }

\xitemenmain{sample text file }

\xitemxitemmain{sample text file }

\xitemenmain{sample text file }

\xitemxitemmain{sample text file }

\xitemxitemmain{sample text file }

\xitemxitemmain{No Lang sample text file }

\end{document}
